release.sh

```diff
@@ -19,6 +19,8 @@ fi
 ################################
 
 # 1. Update file `VERSION` with the numeric version, e.g. `0.3.1`.
+#    Note that VERSION file is required for building tool promu:
```

Comment:
Nit, remove "Note that", it doesn't add any value.
Write "is required by" instead of "for".

Reply:
Thanks for the wording suggestion to make it precise. Updated.